pandas/tests/io/excel/test_xlsxwriter.py

```diff
@@ -64,6 +64,7 @@ def test_write_append_mode_raises(ext):
             ExcelWriter(f, engine="xlsxwriter", mode="a")
 
 
+
```

Comment:
Remove this blank line, it will not pass code checks (two blank lines between functions)